<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Server</class>
 <widget class="QWidget" name="Server">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>93</width>
    <height>96</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Server</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" alignment="Qt::AlignRight">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Porta:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1" alignment="Qt::AlignLeft">
    <widget class="QSpinBox" name="spinPorta"/>
   </item>
   <item row="1" column="0" colspan="2" alignment="Qt::AlignHCenter|Qt::AlignVCenter">
    <widget class="QPushButton" name="btCon">
     <property name="text">
      <string>&amp;Conectar</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="2" alignment="Qt::AlignHCenter|Qt::AlignVCenter">
    <widget class="QPushButton" name="btDis">
     <property name="text">
      <string>&amp;Desconectar</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
